src/lib/components/sandbox/arena/ArenaActionLog.svelte

```diff
@@ -131,26 +129,28 @@
 		}
 
 		return `${text}<br/><br/>`;
-	}
+	};
 
 	const meleeAttackActionLogText = (action: GamePieceAction): string => {
 		let text = '';
-		const attacks = action.actionData.resolvedAttacks;
+		const attack = action.actionData.resolvedAttack;
 		const attackingPiece = action.gamePiece;
 		const targetPiece = action.actionData.targetGamePiece;
-		if (!targetPiece || !attacks) return '';
+		if (!targetPiece || !attack) return '';
 
 		const attackingPlayerUnit = attackingPiece.playerUnit;
 		const targetPlayerUnit = targetPiece.playerUnit;
 		const attackingPlayerKey = attackingPiece.gamePlayer.player.minaPublicKey;
 		const targetPlayerKey = targetPiece.gamePlayer.player.minaPublicKey;
 		const attackingPlayerColor = colorByPlayerKey[attackingPlayerKey] || 'black';
-		const targetPlayerColor = targetPlayerKey ? colorByPlayerKey[targetPlayerKey] || 'black' : 'black';
-		
+		const targetPlayerColor = targetPlayerKey
+			? colorByPlayerKey[targetPlayerKey] || 'black'
+			: 'black';
+
 		const attackingPieceTitle = `<span style="color: ${attackingPlayerColor}">${attackingPlayerUnit.name} (${attackingPlayerUnit.unit.name})</span>`;
 		const targetPieceTitle = `<span style="color: ${targetPlayerColor}">${targetPlayerUnit.name} (${targetPlayerUnit.unit.name})</span>`;
 
-		text += `${attackingPieceTitle} strikes x${attacks.length} at ${targetPieceTitle}<br/>`;
+		text += `${attackingPieceTitle} strikes at ${targetPieceTitle}<br/>`;
```

Comment:
I can make a follow-up PR after this which groups these attack actions by the acting GamePiece and summarizes them together, so that we can still have this aggregated view (e.g. "<attacker> made 5 attacks against <target>")